pdns/dnsdistdist/configure.ac

```diff
@@ -56,6 +54,8 @@ AS_IF([test "x$enable_hardening" != "xno"], [
   AC_LD_RELRO
 ])
 
+PDNS_CHECK_OS
+
 PDNS_ENABLE_SANITIZERS
 
 PDNS_CHECK_PANDOC
```

Comment:
`PDNS_CHECK_CLOCK_GETTIME` was in there already (line 17) - because on some systems it needs `-lrt` I think, and the symlink is already there because of it.